pages/linux/quickemu.md

```diff
@@ -0,0 +1,33 @@
+# quickemu
+
+> Build and manage highly optimised desktop virtual machines quickly.
+> See also: `quickget`, for preparing VM configurations.
+> More information: <https://github.com/quickemu-project/quickemu>.
+
+- Create and run a VM from a configuration file:
+
+`quickemu --vm {{path/to/file.conf}}`
+
+- Do not commit any changes to disk/snapshot but write any changes to temporary files:
+
+`quickemu --status-quo --vm {{path/to/file.conf}}`
+
+- Start the VM in full screen mode (<Ctrl> + <Alt> + f to exit) and select the display backend ('sdl' by default):
+
+`quickemu --fullscreen --display {{sdl|gtk|spice|spice-app|none}} --vm {{path/to/file.conf}}`
+
+- Select a virtual audio emulation and create a desktop shortcut:
```

Comment:
```suggestion
- Select a virtual audio emulation device and create a desktop shortcut:
```
Don't know if it's the correct term here, feel free to reword it.

Reply:
```suggestion
- Select virtual audio emulation and create a desktop shortcut:
```
Maybe this?

Reply:
```suggestion
- Select a virtual audio device to emulate and create a desktop shortcut:
```
@kbdharun what about this?

Reply:
Sure, LGTM.